packages/compass-aggregations/.depcheckrc

```diff
@@ -3,3 +3,5 @@ ignores:
   - '@mongodb-js/tsconfig-compass'
   - '@mongodb-js/tsconfig-compass'
   - 'mongodb-compass'
+ignore-patterns:
```

Comment:
nice, had no idea this was a thing! isn't `.gitignore` honoured?

Reply:
Tbh I'm not sure if it should (would make a lot of sense!), I'll double check, but for me not ignoring dist explicitly here started causing issues for some reason, maybe because we have `dist` ignored at the root level and not at the package level?